resources/js/store/modules/session.js

```diff
@@ -68,7 +68,7 @@ const mutations = {
 
   setCurrentUser (state, currentUser) {
     state.currentUser = currentUser;
-    PermissionService.setCurrentUser(state.currentUser);
+    PermissionService.setCurrentUser(state.currentUser, false);
```

Comment:
On logout when the current user is set on the permission service there shouldn't be fired a recompution of the permissions in the can and cannot components, since they will try to remove components due to missing permissions and in the same time they will be removed since the user isn't authenticated any more, what leads to errors.